es7/2017-01/jan-24.md

```diff
@@ -0,0 +1,1448 @@
+
+JHD: Why not make the notes the official minutes?
+
+AWB: The notes are appended to the minutes, but there is more in the minutes.
+
+IS: Correct.
+
+JHD: Do those on our own without relying on ECMA?
+
+IS: To my understanding the Technical Notes of a Meeting are for you, and one can access it via this address.
+
+WH: ECMA act as librarians for a long-term record. Our many past attempts have not lasted more than a few years; see, for example, what happened to our ecmascript.org.
+
+IS: Exactly. We have a long-term archival obligation. We are trying to archive the most important documents of the standardization process and this in a way that is accessible and understandable to every ECMA member. Many of them simply do not know the present and past TC39 tools
+
+AWB: There is room to work with ECMA to improve the infrastructure for taking notes--if someone wants to do that, everyone would probably appreciate it.
+
+I.S.: We could upload the ECMA minutes on the GitHub to the TC39 Reflector part, would that help?
+
+WH: I was able to eventually somehow get the minutes.
+
+AWB: We'll send this around to get token votes; we can make a PDF of it
+
+#### Side Discussion
+
+WH: What's happening with es-discuss? I'm getting bounces.
+
+AWB: The list administrators of this are Brendan Eich and myself. If someone else wants to be an administrator on that list, I am sure Brendan and I would be happy about that. There isn't a lot of work but occasionally something does come in.
+
+I.S.: I think this is an example for another TC39 tool, that was popular for a few years, and maybe now is slowly disappearing. It is a real challange for us, how to archive all this for long-term.  
+
+## ECMA262 Status Updates--countdown to ES2017
+
+(Brian Terlson)
+
+BT: The key dates for our schedule are, at the end of this month, on the 31st, we will stop taking any normative updates besides bugfixes. we can fix bugs but we can't add features. Correct me if I'm wrong but the only proposal I'm aware of that is pushing for stage 4 is shared array buffers. Are there other proposals if I need to be tracking?
+
+DE: There may be some for 402
+
+AK: Do you mean big proposals, or do bugs count?
+
+BT: Any needs-consensus PRs after January 31st becomes a part of ES2018 (if at all). 
+BT: For those new to the committee, we're not a technical committee, we're a royalty free task group (of the technical committee). All that means is that there is a royalty free commitment to the content we create in the specification. If you would like to patent something that is in ECMA or if you think you have a patent that is pertinent to something we are standardizing in this version. You should talk to your lawyers and potentially opt out of the royalty free agreement. you'd be the first person to do this. We've never had an opt out. Talking to Istvan, I'm not sure we even know what happens.
```

Comment:
There should be one more line break before this line or alternatively you can remove `BT: ` in line 60.